src/Microsoft.CodeAnalysis.Analyzers/CSharp/MetaAnalyzers/CSharpReportDiagnosticAnalyzer.cs

```diff
@@ -33,6 +33,23 @@ protected override IEnumerable<SyntaxNode> GetArgumentExpressions(InvocationExpr
 
                 return null;
             }
+
+            protected override SyntaxNode GetPropertyGetterBlockSyntax(SyntaxNode declaringSyntaxRefNode)
+            {
+                if (declaringSyntaxRefNode is AccessorDeclarationSyntax accessor &&
+                    accessor.Body == null &&
+                    accessor.ExpressionBody == null)
+                {
+                    // Walk up to the property initializer.
+                    var propertyDecl = accessor.FirstAncestorOrSelf<PropertyDeclarationSyntax>();
```

Comment:
Why not `accessor.Parent`?

Reply:
`accessor.Parent`, that seems to accessor list syntax. I felt this was much more safer.

Reply:
Derp, yes true.